Tagger/src/imageTag.java

```diff
@@ -61,10 +62,32 @@ private static void GetTagWhitelist() {
 					s = br.readLine();
 					while (s!=null) {
 						String newtag = s.trim().toLowerCase();
-						tag_whitelist.put(newtag,true);
-						System.out.println("Read in whitelisted tag: "+newtag);
+						String[] combtag;
+						List<String> listofsubs = new ArrayList<String>();
+						if (s.contains(": ")); {
+							combtag = s.split(": ");
+								for (int i=1;i<combtag.length; i++) {
+									tag_whitelist.put(combtag[i].trim().toLowerCase(),true);
+									System.out.println("Read in whitelisted tag: "+combtag[i]);
+									subtaglist.put(combtag[i],listofsubs);

```

Comment:
In lieu of other changes, no more need to add to a list, just insert the tag directly as the value.